lib/inspec/resources/mssql_session.rb

```diff
@@ -53,7 +53,7 @@ def initialize(opts = {})
     end
 
     def query(q) # rubocop:disable Metrics/PerceivedComplexity
-      escaped_query = q.gsub(/\\/, '\\\\').gsub(/"/, '\\"').gsub(/\$/, '\\$')
+      escaped_query = q.gsub(/\\/, '\\\\').gsub(/"/, '""').gsub(/\$/, '\\$')
```

Comment:
I believe this is right for SQL... but this type of change _should_ have a test behind it if at all possible. I just saw your comment about your ruby experience so I'm gonna approve and try to follow up on this with the test (or at least file an issue for it).